spec/views/companies/_map_companies.haml_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+class TempCompanyHelper
+  include CompaniesHelper
+end
+
+RSpec.describe "company maps partial: _map_companies.html.haml" do
+
+  it "map markers are raw json (not escaped)" do
+
+    test_companies = []
+    test_companies << create(:company)
```

Comment:
Also, to be consistent with naming convention, this file should be named:
```
_map_companies.html.haml_spec.rb
```
It still works regardless, but its nice to be consistent :)